### PR TITLE
Add finance roles

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: roles.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 **Table of Contents**
 - [Cultivator](#cultivator-aka-meeting-chair)
+- [Finance (External)](#finance-external)
+- [Finance (Internal)](#finance-internal)
```
Comment (benhylau) [inline roles.md:6]: @hyphacoop/member-owners anyone has better names for these?

**Comment by benhylau** [comment]
@YurkoWasHere I need this merged https://github.com/hyphacoop/handbook/pull/64 otherwise this won't pass.

File: roles.md [hunk 2]
```diff
@@ -34,6 +36,72 @@ meetings (summarized above).
...
+**Aim:** Fulfill financial responsibilities to employees and contractors, and adhere to associated legal and tax obligations.
+
+**Tasks:**
+- run payroll for each pay period
+	- [verify timesheets](https://handbook.hypha.coop/guides.html#timesheets)
+	- [approve reimbursement of expenses](https://handbook.hypha.coop/guides.html#expense-reimbursement)
+	- [prepare payroll](https://handbook.hypha.coop/guides.html#running-payroll)
+	- ensure payroll taxes are remitted, manually file a nil remittance if a pay period is skipped
+	- archive [Employee Payroll sheet](http://link.hypha.coop/payroll) and [processed payroll records](https://drive.google.com/drive/u/0/folders/1wWo9KqNwWdUK5d-jkApV3id_Y_dpftT9)
+- [onboard employees and contractors](https://handbook.hypha.coop/guides.html#onboarding-checklist), keeping appropriate records
```
Comment (dcwalk) [inline roles.md:89]: Don't think this should be a task for finance. There is only a small part that is finance

File: roles.md (hunk 1 above)
Comment (dcwalk) [reply roles.md:6]: Accounts Payable/Receivable (Bookkeeper? Analyst? Auditor?)
Payroll Coordinator

Comment (benhylau) [reply roles.md:6]: Permaculture themed ones? 😄

File: roles.md (hunk 2 above)
Comment (benhylau) [reply roles.md:89]:
```suggestion
- assist Operations WG in [employee and contractor onboarding](https://handbook.hypha.coop/guides.html#onboarding-checklist) and keep appropriate records
```

File: roles.md [hunk 3]
```diff
@@ -34,6 +36,72 @@ meetings (summarized above).
...
+- adhere to legal requirements around [keeping records](https://www.canada.ca/en/revenue-agency/services/tax/businesses/topics/keeping-records.html) and [handling of Social Insurance Number](https://www.canada.ca/en/employment-social-development/services/sin/reports/code-of-practice/section-3.html), and [our sensitive data practices](https://handbook.hypha.coop/guides.html#sensitive-data)
+
+**Authority:**
+- access to sensitive data on employee records
```
Comment (YurkoWasHere) [inline roles.md:93]: We define authority for `external` to move money but done define it for `internal`

Perhaps specify authority is inherited from external? Is there anything in external we don't want to put on internal?

Additionally to the external role authorities
If TransferWise for example has a balance of > 10,000 and we need to move it into the main operating account. 
```suggestion
- all authorities granted to Finance (Internal)
- move balances into the main operating account greater the $10,000
- access to sensitive data on employee records
```
```

Comment (benhylau) [reply roles.md:93]: I specifically want only the external role to move money, so there will not be a case where two ppl are moving money without talking. If internal person needs to move money, aside from payroll, they need to ask the WG.

File: roles.md (hunk 1 above)
Comment (benhylau) [reply roles.md:6]: Finance (External) -> Financial Resource Coordinator
Finance (Internal) -> Worker Caretaker

Comment (dcwalk) [reply roles.md:6]: These feel pretty confusing/not informative of what they do (nor really capturing the permaculture/mycology spirit?) 

Can we bring a hsortlist fo the all hands and aim to get feedback there?

Comment (benhylau) [reply roles.md]: Finance set a deadline to sort these names by weekend, so I am going with `Finance Steward` and `Payroll Coordinator`. We can make future revisions as needed. Don't think it's worthwhile to bring the naming into all hands.

**Pull request merged**